src/platform/testing/e2e/cypress/support/index.js

```diff
@@ -1,11 +1,11 @@
 import { join } from 'path';
-
 import '@testing-library/cypress/add-commands';
 import 'cypress-axe';
 import 'cypress-plugin-tab';
-
 import './commands';
 
+const addContext = require('mochawesome/addContext');
```

Comment:
Mixing ES imports and CJS requires here. Could be consistent by using `import`:

```suggestion
import addContext from 'mochawesome/addContext';
```

I would also place this above `import './commands';` since this is a third-party package.

Reply:
Woops! I missed that. Thanks Euguene.